diff --git a/app/controllers/super_admin/application_controller.rb b/app/controllers/super_admin/application_controller.rb
index 5b04fbb..d6967d8 100644
--- a/app/controllers/super_admin/application_controller.rb
+++ b/app/controllers/super_admin/application_controller.rb
@@ -12,6 +12,7 @@ class SuperAdmin::ApplicationController < Administrate::ApplicationController
   helper_method :render_vue_component, :settings_open?, :settings_pages
   # authenticiation done via devise : SuperAdmin Model
   before_action :authenticate_super_admin!
+  before_action :set_locale
 
   # Override this value to specify the number of elements to display at a time
   # on index pages. Defaults to 20.
@@ -45,4 +46,8 @@ class SuperAdmin::ApplicationController < Administrate::ApplicationController
     # rubocop:enable Rails/I18nLocaleTexts
     redirect_back(fallback_location: root_path)
   end
+
+  def set_locale
+    I18n.locale = (ENV['DEFAULT_LOCALE'] || I18n.default_locale).to_sym
+  end
 end
diff --git a/app/helpers/super_admin/navigation_helper.rb b/app/helpers/super_admin/navigation_helper.rb
index 5fca3fa..405717b 100644
--- a/app/helpers/super_admin/navigation_helper.rb
+++ b/app/helpers/super_admin/navigation_helper.rb
@@ -9,7 +9,7 @@ module SuperAdmin::NavigationHelper
     end
 
     # Add general at the beginning
-    general_feature = [['general', { 'config_key' => 'general', 'name' => 'General' }]]
+    general_feature = [['general', { 'config_key' => 'general', 'name' => 'Geral' }]]
 
     general_feature + features.to_a
   end
